src/anonymous-purchase-consequences-screen/AnonymousPurchaseConsequencesScreen.tsx

```diff
@@ -0,0 +1,100 @@
+import {Support} from '@atb/assets/svg/mono-icons/actions';
+import {Phone} from '@atb/assets/svg/mono-icons/devices';
+import {Receipt} from '@atb/assets/svg/mono-icons/ticketing';
+import {ThemeText} from '@atb/components/text';
+import {ThemeIcon} from '@atb/components/theme-icon';
+
+import {Consequence} from './components/Consequence';
+import {StyleSheet, useTheme} from '@atb/theme';
+import {getStaticColor, StaticColorByType} from '@atb/theme/colors';
+import {AnonymousPurchasesTexts, useTranslation} from '@atb/translations';
+import React from 'react';
+import {ArrowRight} from '@atb/assets/svg/mono-icons/navigation';
+import {OnboardingFrame} from '@atb/onboarding-frame';
+
+const themeColor: StaticColorByType<'background'> = 'background_accent_0';
+
+type Props = {
+  onPressContinueWithoutLogin: () => void;
+  onPressLogin?: () => void;
+};
+
+export const AnonymousPurchaseConsequencesScreen = ({
```

Comment:
Should still be named `AnonymousPurchaseConsequencesScreenComponent`.
When a component is detached from the navigation object it is no longer a screen, but a reusable screen component.
https://github.com/AtB-AS/docs-private/blob/main/kundevendt-development-guidelines.md#screen-reuse

Reply:
After reading [this comment](https://github.com/AtB-AS/mittatb-app/pull/4064#discussion_r1406133743) a bit too quickly and broadly, I excluded `Component` from both this one and `OnboardingScreen` 😅 Will fix!